mylyn.github/org.eclipse.mylyn.github.ui/META-INF/MANIFEST.MF

```diff
@@ -30,7 +30,8 @@ Export-Package: org.eclipse.mylyn.internal.github.ui;version="6.2.1";x-internal:
  org.eclipse.mylyn.internal.github.ui.issue;version="6.2.1";x-internal:=true,
  org.eclipse.mylyn.internal.github.ui.pr;version="6.2.1";x-internal:=true
 Bundle-ActivationPolicy: lazy
-Import-Package: org.apache.commons.lang3;version="3.13.0",
+Import-Package: org.apache.commons.httpclient;version="[3.1.0,4.0.0]",
```

Comment:
Why is httpclient added?

Reply:
Good catch! This can be removed. Left over from another way to fix this problem.

Reply:
Thank goodness because that thing is really ancient:

https://repo1.maven.org/maven2/commons-httpclient/commons-httpclient/3.1/